cmd/vitconfig/vitconfig.go

```diff
@@ -18,15 +18,16 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/input-output-hk/jorvit/pkg/vcli"
+	"github.com/input-output-hk/jorvit/pkg/vstation"
```

Comment:
Could use the same path as the `internal` modules below; I don't see how these modules are special now that they are in tree.

Reply:
Well, vitcli was a "public" API wrapper around the servicing station cli. In Go, whatever public api should go in the pkg folder, hence the decision of keeping it there. If we are gonna maintain it and it is needed externally is better we follow the go styles for this kind of stuff.